### ISSUE TITLE
[Bug]: Laravel command not working after PHP version change

### ISSUE THREAD

**Issue by MrBMT** [beyondcode/herd-community#638]
### Platform

macOS

### Operating system version

macOS Sonoma 14.4.1 (23E224)

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.5.0 (Build: 23)

### PHP Version

PHP 8.1.26 (cli) (built: Dec 21 2023 08:51:01) (NTS)

### Bug description

On a brand new Mac fresh out the box, so has never had Herd, Valet or Homebrew installed, install the latest version of Herd.

It will install PHP 8.3 by default, go to settings and download and install PHP 8.1.

Switch to PHP 8.1 globally by selecting `Use PHP 8.1` from the Herd menu, and try to run the `laravel` command.

It throw an error about an incompatible version of PHP being used, as composer/laravel are trying to use a version for a newer version of PHP than the one you're using now using.

My expected behaviour here would be that if the PHP version is changed globally, any dependancies to run the `laravel` command should also be updated to ensure the command works.

### Steps to reproduce

Install Herd
Add PHP 8.1 via Settings -> PHP and switch to using it globally via the Herd menu
Try to run `laravel` command
Laravel will not run, and composer will say that you're not using a required version of PHP

### Relevant log output

```shell
laravel

Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.2.0". You are running 8.1.26. in phar:///Users/owner/Library/Application Support/Herd/bin/laravel/vendor/composer/platform_check.php on line 24
```

**Comment by jrd-lewis** [comment]
I assume that, under the hood, Herd uses the laravel/installer package which does require PHP 8.2 or later to use. If that's the case, then this may be expected behavior instead of a bug.

**Comment by MrBMT** [comment]
Ah yes, the latest version of the laravel installer does indeed require PHP 8.2.

In that case, ideally the `laravel` command should be moved outside of the added root `$PATH` directory so you don't accidentally end up trying to run it with an incompatible PHP version (e.g. it could be moved to a sub-folder within it).

Just a thought, but then maybe following on from the `herd tinker`, `herd php` etc. commands, a `herd laravel` command could also be added that then runs a version of laravel installer compatible with that PHP version.

**Comment by mpociot** [comment]
You could always generate new application via the UI, if you explicitly change your global PHP version to something below PHP 8.2
Otherwise, the error message pretty much tells you what the problem is. So you can fix this by running `herd use 8.3`, run `laravel new ...` and then switch back.
I don't see this as a big issue tbh.

**Comment by Groteus** [comment]
> You could always generate new application via the UI, if you explicitly change your global PHP version to something below PHP 8.2 Otherwise, the error message pretty much tells you what the problem is. So you can fix this by running `herd use 8.3`, run `laravel new ...` and then switch back. I don't see this as a big issue tbh.

But if I want to use PHP 8.1 version, I get an error. Why add PHP versions that are not supported to HERD?

`Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.2.0". You are running 8.1.28. in`
